spec/features/searching/search_category_spec.rb

```diff
@@ -0,0 +1,60 @@
+RSpec.feature 'A user performs a search with the search box', js: true, vcr: true, slow: true do
```

Comment:
Try to have one expect per it block, I Think that this could be refactored into multiple tests